.buildkite/pipeline.build.yml

```diff
@@ -652,3 +514,147 @@
     - cat test_shard.txt
     - bazel test --config=ci $(./ci/run/bazel_export_options)
       $(cat test_shard.txt)
+
+- block: Kick off docker image build for release branches
+      if: build.branch =~ /^releases\//
```

Comment:
nit: indent?

not sure if the "block" will be properly picked up though.

because we do not use groups, it will probably block everything after it, not only the docker image building ones.

Reply:
ah ha, I should use group then